src/themes/appletv/theme.css

```diff
@@ -449,12 +449,8 @@ html {
     color: #c33;
 }
 
-.repeatButton-active {
-    color: #4285f4;
-}
-
-.shuffleQueue-active {
-    color: #4285f4 !important;
+.buttonActive {
+    color: #00a4dc;
```

Comment:
`!important` is required to override hover state, iirc
But it might be better to fix hover state.